dev/package.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "vscode-azureextensiondev",
+    "author": "Microsoft Corporation",
+    "version": "0.1.0",
+    "description": "Common dev dependency tools for developing Azure extensions for VS Code",
+    "tags": [
+        "azure",
+        "vscode"
+    ],
+    "keywords": [
+        "azure",
+        "vscode"
+    ],
+    "main": "out/src/index.js",
+    "types": "index.d.ts",
+    "license": "MIT",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Microsoft/vscode-azuretools"
+    },
+    "bugs": {
+        "url": "https://github.com/Microsoft/vscode-azuretools/issues"
+    },
+    "homepage": "https://github.com/Microsoft/vscode-azuretools/blob/master/dev/README.md",
+    "scripts": {
+        "build": "tsc -p ./",
+        "prepack": "tsc -p ./",
+        "compile": "tsc -watch -p ./",
+        "lint": "tslint --project tsconfig.json -e src/*.d.ts -t verbose",
+        "test": "mocha out/test/**/*.js --ui tdd",
+        "prepare": "node ./node_modules/vscode/bin/install"
+    },
+    "dependencies": {},
+    "devDependencies": {
+        "@types/mocha": "^2.2.32",
+        "@types/node": "^10.11.3",
+        "mocha": "^2.3.3",
```

Comment:
Interesting. It doesn't actually expose anything from mocha in its types so technically it's not a dependency.  Everyone using addPackageLintSuites from this package will need to have mocha, but using addPackageLintSuites is not required.  I think I'll leave it as is for now since it seems technically correct, but if you feel strongly I can change.

Reply:
I mean I don't think 'types' matter for npm packages. It's the javascript (aka the functionality) that really matters.

I guess my question is - if I only have 'vscode-azureextensiondev' as a dependency and not 'mocha' and I call addPackageLintSuites - will those tests run?